include/agencies.csv

```diff
@@ -59,7 +59,7 @@ Harry S. Truman Scholarship Foundation,HSTSF
 Institute of Museum and Library Services,IMLS
 Inter-American Foundation,IAF
 International Boundary and Water Commission,USIBWC
-Broadcasting Board of Governors,BBG
+United States Agency for Global Media,USAGM
```

Comment:
@climber-girl - Can you please move this line down so that it fits in alphabetically?